Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -55,13 +55,15 @@ private set
             {
                 rootLayer = value;
                 rootLayer.ReconstructParentsRecursive();
+                LayerMoved.AddListener(rootLayer.UpdateLayerTreeOrder);
```

Comment:
ik snap niet waarvoor dit nodig is, de tree order wordt al intern aangepast, en hierboven zag ik een fuctie die de subtrees opvraagt door ze actief te berekenen